templates/tfengine/components/app/cicd/variables.tf

```diff
@@ -0,0 +1,36 @@
+{{- /* Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */ -}}
+
+variable "branch_name" {
+  description = "Branch to trigger on"
+  type        = string
+  default     = "^main$"
+}
+
+variable "source_repo_name" {
+  description = "Name of source repo."
```

Comment:
Clarify what the format of this is, and maybe give an example. At the moment it's not clear to me if this is a GitHub repo name, or CSR, and if it's a full URL or not, etc.

Reply:
You're right that the default provides an example. But I still think the description should have something like your second sentence mentioning that it will create a CSR repo with that name in the project.